### PR TITLE
update tensorboards controller to set ISTIO_HOSTS

### PR DESCRIPTION
**Pull request by aubrianna-zhu**
Update ISTIO_HOSTS so that virtual services created by the tensorboard controller can have proper specs

image uploaded here: https://console.cloud.google.com/gcr/images/unity-ai-data-mlp-prd/global/kubeflow/tensorboard-controller?project=unity-ai-data-mlp-prd

### CODE REVIEW

**Comment by nurcan-sonmez** [comment]
same! Looks good to me,but I would trust @vvnpan's and @cypres 's eyes more than mine for this PR :)

File: components/tensorboard-controller/config/base/kustomization.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ configMapGenerator:
   - RWO_PVC_SCHEDULING="True"
   - TENSORBOARD_IMAGE=tensorflow/tensorflow:2.1.0
   - ISTIO_GATEWAY=kubeflow/kubeflow-gateway
+  - ISTIO_HOSTS='*'
```
Comment (vvnpan) [inline components/tensorboard-controller/config/base/kustomization.yaml:11]: this we would need to patch on our side in `ml-platform-kubeflow` right?

**Review by vvnpan — COMMENTED** [review]
otherwise looks good to me, we should start a doc on all the kubeflow patches we've made. When we do version upgrades in the future we can keep track

File: components/tensorboard-controller/config/base/kustomization.yaml (hunk 1 above)
Comment (aubrianna-zhu) [reply components/tensorboard-controller/config/base/kustomization.yaml:11]: yep! updated now

**Comment by aubrianna-zhu** [comment]
> otherwise looks good to me, we should start a doc on all the kubeflow patches we've made. When we do version upgrades in the future we can keep track

do you think it's good enough to look at the diff? https://github.com/kubeflow/kubeflow/compare/master...Unity-Technologies:kubeflow:master
or is a doc preferred?

**Comment by vvnpan** [comment]
> > otherwise looks good to me, we should start a doc on all the kubeflow patches we've made. When we do version upgrades in the future we can keep track
> 
> do you think it's good enough to look at the diff? [kubeflow/kubeflow@master...Unity-Technologies:kubeflow:master](https://github.com/kubeflow/kubeflow/compare/master...Unity-Technologies:kubeflow:master) or is a doc preferred?

yeah let's just look at this for now, outside of scope of your work for this anyways, I might add a ticket in the backlog for this

**Review by vvnpan — APPROVED** [review]

**Pull request merged**